libs/ui-watt/src/index.ts

```diff
@@ -1 +1,3 @@
-export * from './lib/ui-watt.module';
+export { WattModule } from './lib/ui-watt.module';
+export { AutocompleteComponent } from './lib/components/autocomplete/autocomplete.component';
+export { AutocompleteModule } from './lib/components/autocomplete/autocomplete.module';
```

Comment:
Question: Is it correctly understood that these artifacts must be exported here even though they are also part of `WattModule`? The build was failing if they weren't. That's how I understood it from here at least https://angular.io/guide/creating-libraries

Reply:
Yes we need to provide these, as I remember it was since version 9 this is required otherwise the compiler will throw an error. So everything we export in our module, we need export in our index